docs/extensions.rst

```diff
@@ -77,3 +77,46 @@ and treat it as if the client had made a request.
 
 The ASGI server should set the pseudo ``:authority`` header value to
 be the same value as the request that triggered the push promise.
+
+Zero Copy Send
+------------------
+
+Zero copy send allows you to send the contents of a file descriptor to the
+HTTP client with zero copy. ASGI servers that implement this extension will
+provide ``http.response.sendfile`` in the extensions part of the scope::
+
+    "scope": {
+        ...
+        "extensions": {
+            "http.response.sendfile": {},
+        },
+    }
+
+The ASGI framework can call ``sendfile`` by sending a message with
+the following keys. This message can be sent at any time after the
+*Response Start* message but before the final *Response Body* message,
+it can be mixed with ``http.response.body``. It can also be called
```

Comment:
it can -> and can

Reply:
Ok.

Reply:
My English is not good. If there are any grammatical errors, please let me know. Thanks for your review.

Reply:
That's OK then, I will do grammar fixes for you once we're ready to land this!